cansat-gps/tests/main.rs

```diff
@@ -15,29 +15,49 @@ fn gps_last_nmea_returns_none_if_no_msg_received() {
     assert!(gps.last_nmea().is_none());
 }
 
-#[ignore]
 #[test]
 fn gps_last_nmea_returns_msg_if_msg_received() {
     let uart = mock::Serial::new(FIRST_MSG.to_owned());
-    let _ = Gps::new(uart);
+    let mut gps = Gps::new(uart);
+
+    // Read until the first message is fully received (the second element of the tuple returned by read_serial() is true)
+    // So as long as it is 'false', it means that the read operation has not yet been completed, and only when the value becomes 'true', it means that the entire GPS message has been read.
+    // to print _read_data use 'cargo test --package cansat-gps -- --nocapture' command
+    while let Ok((_read_data, false)) = gps.read_serial() {
+        //print!("{}", _read_data);
```

Comment:
I think we should either remove the commented println! Or uncomment them since they are displayed only with --nocapture flag

Reply:
Thanks, I think it's better if I delete that.